.github/dependabot.yml

```diff
@@ -0,0 +1,14 @@
+version: 2
+updates:
+- package-ecosystem: pip
+  directory: "/"
+  schedule:
+    interval: daily
+    time: "10:00"
+  open-pull-requests-limit: 10
+  ignore:
+  - dependency-name: cryptography
```

Comment:
Given that [in-toto](https://github.com/in-toto/in-toto/releases/tag/v1.0.1) and [tuf](https://github.com/theupdateframework/tuf/releases/tag/v0.17.0) have both dropped Python 2 support, we should make a [final Python2 supporting release](https://github.com/secure-systems-lab/securesystemslib/issues/344) and drop this `ignore` statement.

If there are no objections, I'll handle this over the next few days cc @jku @SantiagoTorres @adityasaky

Reply:
Sounds good to me!